pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -70,6 +71,10 @@ type DesiredStateOfWorldPopulator interface {
 	HasAddedPods() bool
 }
 
+var (
+	pvcBeingDeletedMsg = "PVC is being deleted"
```

Comment:
I would make this a typed error so that both the setter and consumer are consistent

Reply:
The check for the scenario is only done in this class.
It seems current way is fine.